### ISSUE TITLE
Staging Review finding: No screen reader announcement for error state

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#110798]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool 2
**Product:** Claim Status Tool
**Feature:** Evidence Requests

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When on a "needed from you" page (eg. test user 171, [employment information](https://staging.va.gov/track-claims/your-claims/600711245/needed-from-you/588663)), if you attempt to submit documents without actually uploading any documents, an error message appears on the file input, "Please select a file first."

However, nothing is announced to screen reader users when that error message appears. If you incorrectly believe you've added a file to upload and then press the button, you just get silence and you're left to guess what happened. Is it still working? Is something broken? Did I do something wrong? Should I press the button again? etc.

<details><summary>Link, screenshot or steps to recreate</summary>

1. Log in using test user 171.
2. Navigate to a "needed from you" page (eg. test user 171, [employment information](https://staging.va.gov/track-claims/your-claims/600711245/needed-from-you/588663)).
3. Activate a screen reader.
4. Press [tab] to move focus ot the "Submit documents for review" button.
5. Press [enter].
6. Note that the inline error message appears with the file input, but no announcement is made by the screen reader.

</details>

## Recommended action

Typically this is handled either through focus management (send keyboard focus to the input in error) or using an aria-live region or `role="alert"`. The file input component has a `role="alert"` that appears to be coded correctly from a quick review, so I'm not sure what's happening here.

I suspect this may need to be escalated to the Design System Team.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Form - File input component](https://design.va.gov/components/form/file-input)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
